<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="IntroPage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="iconLabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="MainPage">
       <widget class="QPushButton" name="analyseButton">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>420</y>
          <width>201</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Analyse</string>
        </property>
       </widget>
       <widget class="QPushButton" name="chooseFile">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>160</y>
          <width>181</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Choose Input File</string>
        </property>
       </widget>
       <widget class="QPushButton" name="chooseOutputFolder">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>250</y>
          <width>181</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Choose Output Folder </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="inputFilePathShow">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>160</y>
          <width>441</width>
          <height>24</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="outputFolderPathShow">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>250</y>
          <width>441</width>
          <height>24</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="LoadPage">
       <widget class="QPushButton" name="goBackToMainPageButton">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>480</y>
          <width>161</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Analyse another dump</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="outputDiplayTextBrowser">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>220</y>
          <width>581</width>
          <height>231</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>80</y>
          <width>571</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QLabel" name="timeLabel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>150</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>00 : 00 : 00</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>150</y>
          <width>91</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Elapsed Time: </string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
